### ISSUE TITLE
Next interview Tuesday May 7

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#122]
Just an FYI, that I will need a working sim ported to sonification wrapper by end of day Friday May 3rd or end of Day Monday May 6th.

Interview is with a consultant to check about force focus alerts and qualitative distance.

**Comment by terracoda** [comment]
I should have another consultant interview later in the week of May 6th as well.

**Comment by terracoda** [comment]
**Confirmed Time for next interview**
- Tuesday May 7th, 4:00 am AKDT / 7:00 am CT / 14:00 CET

Need ported sonified versions that hopefully have #124 and #125 redesigns included.
@zepumph, just do what you can. I can also discuss these changes with the consultant to get feedback.

**Comment by zepumph** [comment]
@terracoda  here you are! https://phet-dev.colorado.edu/html/phet-io-wrapper-sonification/1.0.0-dev.113/phet-io-wrapper-sonification/gravity-force-lab-basics/html/gravity-force-lab-basics-sonification.html?sonificationFile=sonificationOptions&massSound=0&forceSound=0&massBoundaryLimitSound=2&accessibility

**Comment by terracoda** [comment]
I did two sessions with screen reader consultants with the above version (May 7th and May 10th).
